packages/kbn-scout/src/playwright/fixtures/test/validate_tags.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { test as base } from '@playwright/test';
+import { tags } from '../../constants';
+
+const supportedTags = tags.DEPLOYMENT_AGNOSTIC;
+
+export const validateTagsFixture = base.extend<{ validateTags: void }>({
+  validateTags: [
+    async ({}, use, testInfo) => {
+      if (testInfo.tags.length === 0) {
+        throw new Error(`At least one tag is required: ${supportedTags.join(', ')}`);
+      }
+
+      const invalidTags = testInfo.tags.filter((tag: string) => !supportedTags.includes(tag));
+      if (invalidTags.length > 0) {
+        throw new Error(
+          `Unsupported tag(s) found in test suite "${testInfo.title}": ${invalidTags.join(
+            ', '
+          )}. ` + `Supported tags are: ${supportedTags.join(', ')}.`
+        );
+      }
+
+      await use();
+    },
+    { auto: true },
+  ],
+});
```

Comment:
The goal: notify test author about requirement of tags definition and validating its correctness. I also introduces aliases through `tags` variable to minimize spelling error. 

Note: it only work when you run tests in IDE; when you run tests via `npx playwright test --grep <tag>` playwright is doing filtering in advance and no filters/incorrect filters will cause test to be filtered out.

That said, for CI we will need to come up with some script to validate tags being defined. But I still believe there is a benefit of raising error during test development process

Reply:
Example: adding non-supported tag `@xyz` like 

```ts
test.describe(
  'Discover app - saved searches',
  { tag: ['@ess', '@svlSearch', '@svlOblt', '@xyz'] },
  () => {
```

will lead to error

<img width="1209" alt="image" src="https://github.com/user-attachments/assets/884e00f6-a91f-497e-82f5-7e9cc0fb63a7">